web/src/components/apps/AppVersionHistory.jsx

```diff
@@ -1455,8 +1488,9 @@ class AppVersionHistory extends Component {
               isOpen={this.state.showAutomaticUpdatesModal}
               onRequestClose={this.toggleAutomaticUpdatesModal}
               updateCheckerSpec={app?.updateCheckerSpec}
-              semverAutoDeploy={app?.semverAutoDeploy}
+              autoDeploy={app?.autoDeploy}
               appSlug={app?.slug}
+              semVerEnabled={appLicense?.isSemverRequired}
```

Comment:
this should be part of the application response object, we shouldn't be making an extra request to get the license and determine that.

Reply:
in here i mean https://github.com/replicatedhq/kots/blob/a606fcfa00ba79894376f43ab96053c3b11a7c43/pkg/handlers/app.go#L165

Reply:
I'll have to add it to the App Response, but I see there are other license fields in their too. I'll consolidate that.